aten/src/ATen/Dispatch_cuda8_compat.h

```diff
@@ -0,0 +1,76 @@
+#pragma once
+
+#include <ATen/Error.h>
+#include <ATen/Half.h>
+#include <ATen/Type.h>
+
+// Same thing as Dispatch.h but without the anonymous lambdas. This exists to
+// provide compatibility with CUDA 8 which does not allow __device__ lambdas
+// to be defined inside other lambdas.
```

Comment:
Need a reference here from `Dispatch.h` to remind people to update this file too.

Reply:
I'm going to delete this